wolfcrypt/src/random.c

```diff
@@ -1404,7 +1404,7 @@ int wc_InitNetRandom(const char* configFile, wnr_hmac_key hmac_cb, int timeout)
 
     /* create/init polling mechanism */
     if (wnr_poll_create() != WNR_ERROR_NONE) {
-        printf("ERROR: wnr_poll_create() failed\n");
+        fprintf(stderr, "ERROR: wnr_poll_create() failed\n");
```

Comment:
This almost seems like a left over debug message that should instead go to WOLFSSL_MSG()? Maybe @cconlon could comment more on it.

Reply:
yeah, there's actually quite a few of those.  we don't have to fix it right here and now, but we should stick a pin in it.